src/components/Tooltips/CanvasHoverTooltip.tsx

```diff
@@ -90,7 +90,7 @@ export default function CanvasHoverTooltip(props: PropsWithChildren<CanvasHoverT
       return null;
     }
     const globalId = lastHoveredId.globalId;
-    const motionDelta = [motionDeltas[2 * globalId], motionDeltas[2 * globalId + 1]];
+    const motionDelta = motionDeltas.subarray(globalId * 3, globalId * 3 + 3);
```

Comment:
Fixes a bug where the sampled data was totally wrong because motionDeltas had been changed to be 3 values per ID instead of 2.